docs/LoongArch-ELF-ABI-EN.adoc

```diff
@@ -628,3 +628,141 @@ with check 32-bit unsigned overflow
 |Linux, Glibc
 |/lib32/ld-linux-loongarch-ilp32s.so.1
 |===
+
+== Procedure Calling Convention
+
+The LoongArch procedure calling convention passes arguments in registers as much as possible (i.e. Floating-point arguments are passed in floating-point registers and non floating-point arguments are passed in general-purpose registers as much as possible)，and then considers passing arguments on the stack. This is a basic principle of LoongArch procedure calling convention.
+
+The argument registers that passes arguments are,
+
+. Eight floating-point registers `fa0-fa7`, and `fa0-fa1` are also used to return values. They are all used to pass floating-point arguments.
+
+. Eight general-purpose registers `a0-a7`, and `a0-a1` are also used to return values. Generally, they are used to pass fixed-point arguments and floating-point arguments when the floating-point register is not available . And some other uses will be described below.
+Argument bit fields are stored in little endian mode. In addition, the subroutine should ensure that the values of general-purpose registers `s0-s8` and floating-point registers `fs0-fs7` are preserved across procedure calls.
```

Comment:
`s0-s8` probably should be `s0-s9`? Because the `fp/s9` register needs to be preserved whether being treated as frame pointer or not.

There's no mention of frame pointer anywhere in the doc, maybe you should add some of that too.

Reply:
Oops,  I didn't notice that. You are very observant, and thank you for your great suggestion!